.github/workflows/test.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     - run: nix-env -iA nix-build-uncached -f nix/
 
     - name: "nix-build"
-      run: nix-build-uncached --max-jobs 4 -A all-systems-go -build-flags -L
+      run: nix-build-uncached --max-jobs 1 -A tests -build-flags -L
```

Comment:
Why this change?

Reply:
so that we don't cross the storage quota of the builder. I have been trying to limit the number of processes (and limiting the build target) so that there is a chance to (slowly) fill up the binary cache and make some progress.

This will be backed out in the final step.

Reply:
It's gone now :-)